Glideslope.json

```diff
@@ -17303,6 +17316,19 @@
    "glideslope": 3.00256927,
    "receptionRange": 10
  },
+ {
+   "name": "GS",
+   "airportICAO": "KIRK",
+   "identifier": "IIRK",
+   "associatedRunwayNumber": "36",
+   "frequency": 111500,
+   "elevation": 965,
+   "latitude": 40.08425745,
+   "longitude": -92.54442726,
+   "bearing": 0.48,
+   "glideslope": 3.00048,
```

Comment:
Should be 3.0000048 because you get 3.00 from the angle at 000.48 from the bearing.

Reply:
Should be all good now, thanks @velocity23!